service/src/test/java/uk/gov/hmcts/reform/fpl/service/config/LookupTestConfig.java

```diff
@@ -14,9 +19,39 @@ public HmctsCourtLookupConfiguration courtLookupConfiguration() {
         return new HmctsCourtLookupConfiguration("example=>Family Court:admin@family-court.com:11");
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public CtscEmailLookupConfiguration ctscEmailLookupConfiguration() {
+        return new CtscEmailLookupConfiguration("Ctsc+test@gmail.com");
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public CafcassLookupConfiguration cafcassLookupConfiguration() {
+        return new CafcassLookupConfiguration("example=>cafcass:FamilyPublicLaw+cafcass@gmail.com");
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public DateFormatterService dateFormatterService() {
+        return new DateFormatterService();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public HearingBookingService hearingBookingService() {
+        return new HearingBookingService();
+    }
```

Comment:
Don't think you need these here

Reply:
it is as it's used in `HmctsEmailContentProvider`

Reply:
Yeah but, they don't require any value parameters passed to them so spring should just be able to create the normal one

Reply:
As long as it is added to the context config or whatever version of it that you were using it should be fine